tests/vm_startup_test.go

```diff
@@ -0,0 +1,123 @@
+package tests
+
+import (
+	"context"
+	"fmt"
+	"os/exec"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	k8sv1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+
+	v1 "kubevirt.io/api/core/v1"
+
+	networkv1 "github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/apis/k8s.cni.cncf.io/v1"
+)
+
+const timeout = 3 * time.Minute
+const pollingInterval = 5 * time.Second
+const testNamespacePrefix = "secondary-test"
```

Comment:
why we need this with the random suffix ?
even if we run on parallel, there will be a namespace per instance as kubevirt is doing

Reply:
It is not a good practice imo, and i less like the way using different namespaces
I prefer that we delete VMI and not wait for them but not delete the NS as we are doing on kubevirt
note that on kubevirt there are some places where we do wait for all resources to gone